impeller/display_list/skia_conversions.h

```diff
@@ -34,10 +34,13 @@ bool IsNearlySimpleRRect(const SkRRect& rr);
 Rect ToRect(const SkRect& rect);
 
 std::optional<Rect> ToRect(const SkRect* rect);
+std::optional<const Rect> ToRect(const flutter::DlRect* rect);
```

Comment:
Are any of these conversions unused now?

Reply:
I guess we still use SkRect and friends for many of the draw methods, and we do need to do the copying allocation until experimental canvas gets further along..

Reply:
Towards the end of this effort I will start aggressively removing these converters. For now I'm not going to sweat each one. When I'm done the entire impeller::skia_conversions package will no longer exist. That will be the real goal.